packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -278,6 +278,7 @@ export default class RestApi {
         'VM-snapshot',
         'VM-template',
         'VM',
+        'SM',
```

Comment:
This will add the `SM` collection to the old REST API.
Instead, insert it into the new REST API: `@xen-orchestra/rest-api`.
Here's what you need:
- Define the "SM" type in `@vates/types/xo.mts`
- Create a controller (sm.controller.mts) that extends `XapiXoController<YourSmType>`
- Expose `getSms` and `getSm` using appropriate decorators. (You can see what's been done for other collections.)

Reply:
Yes, this is already done in a second PR, I wait for this one to be accepted to rebase the second one.

Reply:
https://github.com/vatesfr/xen-orchestra/pull/8687/files